dockerfiles/agents/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/hyperledger/aries-cloudagent-python:py3.12-1.0.0rc4
+FROM ghcr.io/hyperledger/aries-cloudagent-bbs:py3.12-1.0.0
```

Comment:
Do we use BBS for anything?
It sounds like we support BBS for JSONLD credentials 🤔 
But JSONLD might not be functional yet?

It looks like Hyperledger has been working on reworking/reimplementing BBS to be ARM compatible.
* https://github.com/hyperledger/aries-bbssignatures-rs/issues/8
* https://github.com/mattrglobal/ffi-bbs-signatures/pull/74
* https://github.com/mattrglobal/ffi-bbs-signatures/pull/77

If we can drop BBS then we can enable multi-arch builds on our Agents.
At least until the rework/reimplementation of BBS is done.

Alternatively, we keep ARM disabled for the agents until BBS is ARM compatible, at which point we then enable ARM builds.

@ff137, what do you think?

From a personally self-interested standpoint, I'd really like to get ARM compatibility sooner rather than later.
The only things currently in this repo that aren't ARM compatible are:
* Aca-Py Agents
* Ledger Nodes/Browser

Reply:
> But JSONLD might not be functional yet?
We have signing and verifying of JSON-LD documents, which does need to be reviewed again to ensure everything is working properly.

But, we support BBS for sending `ld_proof` credentials. So, the issuer can have a did:key with BBS method, and send ld_proof credentials specifying their did. We have tests that assert that works. Other than that, we don't really use BBS.

So, we could disable support for BBS in the meantime. Mainly just need to skip the relevant tests. And then re-enable those when BBS is ARM-compatible.